frontend/src/pages/project/index.vue

```diff
@@ -93,7 +94,15 @@ export default {
         }
     },
     watch: {
-        project: 'checkAccess',
+        project: function () {
+            // this.updateProject()
```

Comment:
Why was this commited?

Reply:
good spot, I had local commit, that hadn't pushed, just merged it with the commented out code removal you suggested too.